spec/bracket-matcher-spec.coffee

```diff
@@ -105,12 +105,12 @@ describe "bracket matching", ->
           expectHighlights([8, 42], [8, 54])
 
     describe "when the cursor is destroyed", ->
-      it "updates the highlights to use the editor's last cursor", ->
+      it "updates the highlights to use the editor's first cursor", ->
```

Comment:
:bulb: I think here we want to keep `last cursor`, as the behavior we're testing is independent from how `TextEditor::consolidateSelections` works (https://github.com/atom/bracket-matcher/blob/master/lib/bracket-matcher-view.coffee#L54).